### DISCUSSION TITLE
performance with large data

### DISCUSSION THREAD

**Discussion by SRGajjala** [KevinVandy/material-react-table#16] [Q&A] [answered]
Does it handle large data?

**Comment by KevinVandy** [comment, marked as answer]
yes, check out the virtualized example with 20,000 rows https://www.material-react-table.com/docs/examples/virtualized

**Reply by KevinVandy** [reply]
though improvements can always be made. I had been looking into column virtualization too, for handling extra wide tables, but had to stop working on that when running into problems.

Report any performance issues you find

**Reply by SRGajjala** [reply]
code in sandbox is not working
https://codesandbox.io/s/github/KevinVandy/material-react-table/tree/main/material-react-table-docs/examples/enable-row-virtualization/sandbox?file=/src/TS.tsx

**Reply by KevinVandy** [reply]
I'm about to release a new version today, but everything is currently out of sync for a bit. Will be fixed soon

**Reply by KevinVandy** [reply]
its in sync now
